core/db/migrate/20220426165309_add_available_until_to_spree_products.rb

```diff
@@ -0,0 +1,14 @@
+class AddAvailableUntilToSpreeProducts < ActiveRecord::Migration[5.2]
+  class SpreeProduct < ActiveRecord::Base
+    self.table_name = "spree_products"
+  end
+  def up
+    add_column :spree_products, :available_until, :datetime
+    SpreeProduct.update_all("available_until = discontinue_on")
+  end
+
+  def down
+    SpreeProduct.update_all("discontinue_on = available_until")
```

Comment:
This should not be needed if they are in sync, right?

Reply:
True, removed!